docs/en/lifecycle.rst

```diff
@@ -0,0 +1,252 @@
+.. include:: ../common/common_definitions.rst
+
+.. _lifecycle.rst:
+
+Wallet Instance and Digital Credential Lifecycle
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+A Credential Issuer is responsible for creating and issuing Digital Credentials, as well as managing their lifecycle and validity status. 
+
+An Authentic Source is the entity responsible for the management and provisioning of User's attributes to Credential Issuers. 
+There is a relationship between the lifecycle of the attributes managed by the Authentic Source and the Digital Credential lifecycle
+managed by the Credential Issuer. Indeed, one of the reasons for revocation or suspension of Digital Credentials is the update/revocation or 
+suspension of the attributes contained in the Digital Credential. In IT Wallet, the provisioning of User's attributes and the notification of 
+updates or changes in the state of the attributes are exchanged using the PDND infrastructure (see relative sections for more details).
+
+Finally, a Wallet Provider is in charge of the implementation and provision of Wallet Instances also handling their entire lifecycle. 
+
+In this section, state machines are presented to explain the Wallet Instance and Digital Credential states and their transitions and relations.
+
+.. note::
+
+  We distinguish PID from (Q)EAA, as its issuance effects on the Wallet Instance's lifecycle, and additionally its revocation may have potential impacts on (Q)EAAs. 
+  When the distinction between PID and (Q)EAA is not needed, the term Digital Credential is used.
+
+.. note::
+
+  In the current version of ARF v1.4.1, two types of attestations have been introduced: Wallet Trust Evidence (WTE) and Wallet Instance Attestation (WIA). The first to prove that the keys used for key 
```

Comment:
```suggestion
  In the current version of `ARF`_, two types of attestations have been introduced: Wallet Trust Evidence (WTE) and Wallet Instance Attestation (WIA). The first to prove that the keys used for key 
```

the version of the ARF in use and referenced in this documentation is the one defined in the referenced standards. We really do not have to disseminate weak or static references within the specs

Reply:
I would like to have the alignments with the IA terminology too- Otherwise we will use a separate PR within milestone 1.0 to have this alignment